src/main/java/uk/gov/hmcts/darts/usermanagement/service/impl/SecurityGroupServiceImpl.java

```diff
@@ -177,7 +179,38 @@ void updateSecurityGroupEntity(SecurityGroupPatch securityGroupPatch, SecurityGr
     }
 
     private void patchSecurityGroupUsers(SecurityGroupPatch securityGroupPatch, SecurityGroupEntity securityGroupEntity) {
```

Comment:
A subtle variation of AC3 is a PATCH request with:
```json
{
   "user_ids": []
}
```
This is used when we wish to remove all (non-system) users from the group.

I think with the implementation here, it would also remove system users. Could you check that please?

P.S I have added AC4 onto the ticket to cover this scenario.

Reply:
good spot. updated to comply with AC4